### ISSUE TITLE
untimeError: set_sizes_and_strides is not allowed on a Tensor created from .data or .detach().

### ISSUE THREAD

**Issue by sysu19351098** [DSE-MSU/DeepRobust#91]
When calling the adversary.generate function
